### PR TITLE
 Omit NullableContextAttribute from ECMAXML

### PR DESCRIPTION
**Pull request by v-fuquanli**
Work items : https://ceapex.visualstudio.com/Engineering/_workitems/edit/949474

**Before update**
image
image

**After update**
image
image

### CODE REVIEW

**Comment by v-fuquanli** [comment]
https://ceapex.visualstudio.com/Engineering/_build/results?buildId=1868406&view=results

**Review by huangmin-ms — DISMISSED** [review]

File: NuGet.config [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!-- used to store credentials -->
+  <!-- Used to specify which one of the sources are active -->
+  <!-- Used to disable package sources  -->
+  <!--
+  Used to specify default API key associated with sources.
+  See: NuGet.exe help setApiKey
+  See: NuGet.exe help push
```
Comment (huangmin-ms) [inline NuGet.config:9]: Remove this file.

**Review by huangmin-ms — APPROVED** [review]

**Comment by v-fuquanli** [comment]
https://ceapex.visualstudio.com/Engineering/_build/results?buildId=1883961&view=results

**Comment by v-fuquanli** [comment]
https://github.com/dotnet/dotnet-api-docs/compare/6fc2cdac747df5fe927ec708e41d21866d73cb80...ef7985ab5341c6ce25e969035e919add9486c19c/

**Pull request merged**